sys/log/Makefile.include

```diff
@@ -16,4 +16,6 @@ endif
 
 ifneq (,$(filter log_color,$(USEMODULE)))
   USEMODULE_INCLUDES += $(RIOTBASE)/sys/log/log_color
+  BASELIBS += log.module
+  SRC += log_color.c
```

Comment:
Can we use the default mechanism of `SUBMODULES` here and not some hand-spun solution, please?

Reply:
yes